src/app/components/core/settings/settings.component.html

```diff
@@ -33,8 +33,11 @@
           </div>
           <div class="column">
             <label class="label is-small is-fullwidth">Port</label>
-            <input id='service_port' class="input" type="number" appNumberOnly min="0" maxlength="4" placeholder="8081"
-              [ngModel]="servicePort">
+            <input name="service_port" id='service_port' class="input" type="number" [minValue]="0" [maxValue]="65535" placeholder="8081"
+              [ngModel]="servicePort" #service_port="ngModel">
```

Comment:
what is `#service_port="ngModel"` next to `[ngModel]="servicePort"`

Reply:
To get the value of `port` in local template variable, I have used it. We require this variable to show error message.